mmv1/third_party/terraform/website/docs/guides/google_project_service.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+page_title: "User Guide for google_project_service"
+description: |-
+  An advanced user guide for the google_project_service resource
+---
+
+# User Guide - google_project_service
+
+## Enabling Multiple Services in a Config
```

Comment:
Maybe add a note to this section about batching? 

> Even though `google_project_service` represents a single resource, the Google provider will batch multiple changes into a single request when possible.

Reply:
(this is covered below already, though)

Reply:
added a note